platformio.ini

```diff
@@ -7,7 +7,7 @@
 lib_deps_shared =
   https://github.com/hytech-racing/shared_firmware_systems.git#
   https://github.com/hytech-racing/shared_firmware_types.git#
-  ; https://github.com/ssilverman/QNEthernet#v0.26.0
+  https://github.com/ssilverman/QNEthernet#v0.26.0
```

Comment:
We don't want QNEthernet in the common libdeps, we only want them in the teensy41 (on-hardware) environment's libdeps